.sauce/apif.yml

```diff
@@ -0,0 +1,21 @@
+apiVersion: v1alpha
+kind: apif
+sauce:
+  region: us-west-1
+  metadata:
+    tags:
+      - e2e
+    build: "$BUILD_ID"
```

Comment:
Does this metadata even work with APIT?

Reply:
Tags doesn't work, but build will. 

So yah, the config is a bit of a mixed bag right now. The `sauce` object has a bunch of shared props (build, tunnel), but a few will be ignored (tags, concurrency).